### PR TITLE
Warn the user if the minimum coverage is set greater than 100

### PR DESCRIPTION
**Pull request by belfazt**
I was trying something out, and I thought that it would be funny/useful to point this out to the user

### CODE REVIEW

File: lib/simplecov/configuration.rb [hunk 1]
```diff
@@ -288,6 +290,10 @@ def add_group(group_name, filter_argument = nil, &filter_proc)
 
   private
 
+    def minimum_possible_coverage_exceeded(method_name)
+      warn "The coverage you set for #{method_name} is greater than 100%"
```
Comment (PragTob) [inline lib/simplecov/configuration.rb]: I think we should rather call this `coverage_option` or something (still doesn't sound great) - _method_name_ makes me think we're doing fancy metaprogramming stuff :)

File: spec/configuration_spec.rb [hunk 2]
```diff
@@ -33,5 +33,19 @@
         expect(config.tracked_files).to be_nil
       end
     end
+
+    describe "#minimum_coverage" do
+      it "warns you about your usage" do
+        expect(config).to receive(:warn).with("The coverage you set for minimum_coverage is greater than 100%")
+        config.minimum_coverage(100.01)
```
Comment (PragTob) [inline spec/configuration_spec.rb]: Thanks for the tests! We're missing a negative test though. Aka 100.0 doesn't trigger a warning (current tests would pass if we just warned all the time it seems like to me)

**Comment by PragTob** [comment]
Ugh just saw the JRuby failure, I'd guess/hope that it's not related I'll rerun that and otherwise will have to do some testing.

**Comment by belfazt** [comment]
Yeah, I don't know what could've gone wrong with JRuby

**Comment by PragTob** [comment]
failure in JRuby is persistent... I'll open a mock PR without changes to see if that also fails and then try to fix it. Sorry for the hassle :(

**Comment by belfazt** [comment]
It seems that rvm is not identifying `9.1.8.0`?

**Comment by PragTob** [comment]
9.2.8.0 yeah, my bad as charlie commented it's fixed there I just tried it but it's not released. I set the JDK version instead. Works, I'll be brave and emrge this :)

**Comment by belfazt** [comment]
Ah, sorry, hehe. Thank you very much!

**Pull request merged**